### PR TITLE
Bump `spine-base`

### PR DESCRIPTION
**Pull request by yevhenii-nadtochii**
This PR picks up the latest changes from `base` module.

### CODE REVIEW

**Comment by yevhenii-nadtochii** [comment]
@armiol @alexander-yevsyukov PTAL

File: version.gradle.kts [hunk 1]
```diff
@@ -32,4 +32,4 @@
  *
  * For dependencies on Spine SDK module please see [io.spine.dependency.local.Spine].
  */
-val protoDataVersion: String by extra("0.69.7")
+val protoDataVersion: String by extra("0.69.8")
```
Comment (armiol) [inline version.gradle.kts:35]: If the API has any breaking changes (has it? considering the changes in `options.proto`?), this should be `0.70.0`.

**Review by armiol — APPROVED** [review]
@yevhenii-nadtochii please consider my comment. Otherwise, LGTM.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM

**Pull request merged**